### PR TITLE
update bazelrc

### PR DESCRIPTION
**Pull request by fansehep**
Signed-off-by: fan <yfan3763@gmail.com>

### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary: update ```.gitignore```.

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -121,8 +121,7 @@ thirdparties/rocksdb/rocksdb/
 thirdparties/rocksdb/*log
 thirdparties/rocksdb/*.tar.gz
 thirdparties/aws/*.tar.gz
-thirdparties/etcdclient/tmp/
-thirdparties/etcdclient/*.h
+thirdparties/etcdclient/*
```
Comment (ilixiaocui) [inline .gitignore:124]: Modifications to files other than `tmp`  under `thirdparties/etcdclient` cannot be ignored.

Comment (fansehep) [reply .gitignore:124]: We usually do not change the contents of this folder, instead when compiling the project, a lot of extra files are created in this directory.

Comment (fansehep) [reply .gitignore:124]: > Modifications to files other than `tmp` under `thirdparties/etcdclient` cannot be ignored.




here you can see it's inconvenient here. And someone use vscode git plugin. so ugly here.

**Comment by fansehep** [comment]
recheck

**Pull request closed without merging**